### DISCUSSION TITLE
[Elasticsearch] lowercase query operators silently ignored in Lucene queries

### DISCUSSION THREAD

**Discussion by vertebarbe** [grafana/grafana#63935] [Data Sources]
Hello, I encountered the same problem detailed in #19605.

I understand that this won't be fixed, but there should be a mention about this behavior in the documentation here:
https://grafana.com/docs/grafana/latest/datasources/elasticsearch/query-editor/

I'm sure many would copy the queries straight out of Kibana or are used to write queries like this... which probably resulted in many falsely working panels already.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82934. Feel free to continue the discussion around this there. Thank you!
